setup.py

```diff
@@ -312,13 +312,11 @@ def get_long_description():
             "Sphinx>=2.1.0,!=3.2.0",
             # Python 3.9 exclusions:
             #
-            # * NumPy installation fails on Python 3.9 on the Ubuntu Xenial
-            #   system used by Travis CI.
-            # * PySide2 is installable but also currently not working
+            # * PySide2 is installable but currently not working
```

Comment:
this is tangential but i just tried creating a python 3.9 venv and i was able to successfully install pyside2 with traits/traitsui/pyface and run the traits testsuite.

Reply:
Ah, nice. I'll give that a try in this PR, since we're already messing with the NumPy part.

Reply:
> I'll give that a try in this PR, since we're already messing with the NumPy part.

And it works! @rahulporuri still okay to merge?

Reply:
Just curious, is it because a newer version pyside2 is made available to work on Python 3.9?

Reply:
@kitchoi Not 100% sure, but I think the last time I was trying to make this work it was with a Python 3.9 pre-release, so working PySide2 wheels didn't exist on PyPI. Or possibly the PyPI wheels simply weren't compatible with Travis CI's flavour of Linux, but are compatible with the GitHub Actions flavour.

Reply:
That's good to know. Thank you!